app/workers/scheduler/follow_recommendations_scheduler.rb

```diff
@@ -51,10 +51,7 @@ def perform
 
       redis.multi do |multi|
         multi.del(key(locale))
-
-        recommendations.each do |(account_id, rank)|
-          multi.zadd(key(locale), rank, account_id)
-        end
+        multi.zadd(key(locale), recommendations)
```

Comment:
I think `zadd` expects the array to be (score, value) whereas the `recommendations` array is (value, score)

Reply:
Good catch. I fixed it.